fastlane_bot/tools/optimizer/margpoptimizer.py

```diff
@@ -236,14 +248,15 @@ def optimize(self, sfc=None, result=None, *, params=None):
                 price_estimates_t = self.price_estimates(
                     tknq=targettkn, 
                     tknbs=tokens_t, 
-                    verbose=P("debug"),
+                    verbose=P("debug_pe"),
                     triangulate=True,
                 )
             except Exception as e:
                 if P("verbose") or P("debug"):
                     print(f"[margp_optimizer] error calculating price estimates: [{e}]")
                 price_estimates_t = None
-                raise
+                if not result in [self.MO_PSTART, self.MO_DEBUG]:
+                    raise
```

Comment:
Consider something more informative, for example:
```
assert result in [self.MO_PSTART, self.MO_DEBUG], f"{result}"
```

Reply:
not the same; `raise` re-raises the exception, `assert` raise a different exception
also this is actually not the point here -- I want the code to continue in those cases (which are non-standard execution paths) because they'll return early anyway